File: 30days/02/02.py
#!/usr/bin/env python

meal_cost = float(input().strip())
tip_percent = int(input().strip())
tax_percent = int(input().strip())

total = round(meal_cost * (1 + (tip_percent + tax_percent)/100))

print("The total meal cost is %d dollars." % total)

File: 30days/02/02.rb
#!/usr/bin/env ruby

meal_cost = gets.strip.to_f
tip_percent = gets.strip.to_f
tax_percent = gets.strip.to_f

total = meal_cost * (1 + (tip_percent + tax_percent) / 100)

puts "The total meal cost is #{total.round} dollars."
